src/sql/workbench/contrib/notebook/browser/notebookActions.ts

```diff
@@ -446,81 +446,20 @@ export class AttachToDropdown extends SelectBox {
 			this.setOptions([msgLocalHost]);
 		}
 		else {
-			let connections = this.getConnections(model);
-			this.enable();
-			if (showSelectConnection) {
-				this.loadWithSelectConnection(connections);
+			let connections: string[] = model.context && model.context.title ? [model.context.title] : [];
+			if (connections.length === 0) {
+				connections.push(msgSelectConnection);
 			}
-			else {
-				if (connections.length === 1 && connections[0] === msgAddNewConnection) {
-					connections.unshift(msgSelectConnection);
-				}
-				else {
-					if (!find(connections, x => x === msgAddNewConnection)) {
-						connections.push(msgAddNewConnection);
-					}
-				}
-				this.setOptions(connections, 0);
-			}
-		}
-	}
-
-	private loadWithSelectConnection(connections: string[]): void {
-		if (connections && connections.length > 0) {
-			if (!find(connections, x => x === msgSelectConnection)) {
-				connections.unshift(msgSelectConnection);
-			}
-
-			if (!find(connections, x => x === msgAddNewConnection)) {
-				connections.push(msgAddNewConnection);
+			if (!find(connections, x => x === msgChangeConnection)) {
+				connections.push(msgChangeConnection);
```

Comment:
Huh, is it ever expected that model.context.title would ever be msgChangeConnection ? I don't like doing string searches like this when we can avoid them and it seems kind of odd that we'd need to do this check at all